src/routes/+page.svelte

```diff
@@ -0,0 +1,20 @@
+<style>
```

Comment:
looks good, but let's put our colors in a different file (perhaps `style.css`?) and please test that we can get typescript working